### PR TITLE
LL-8553 - add subAccount playwright

### PR DESCRIPTION
**Pull request by Arnaud97234**
## 🦒 Context (issues, jira)
Playwright - UI test
Add Accounts flow -> token account

https://ledgerhq.atlassian.net/browse/LL-8553

## 💻  Description / Demo (image or video)

- Migrate add sub account test from spectron.
- Receive modal page object
- AccountsPage page object

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

**Review by valpinkman — DISMISSED** [review]

**Review by valpinkman — DISMISSED** [review]

**Review by nabil-brn — CHANGES_REQUESTED** [review]
Recommendations:

- Some indent issues
- Test is a bit flaky, after 4 executions, it failed once
- Please check the screenshots that have been taken (e.g. loading steps are not relevant)
- Split tests as much as you can by features/flow etc. (e.g. you can create a `receive.spec.ts` test file)

File: src/renderer/modals/Receive/steps/StepReceiveFunds.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ const StepReceiveFunds = ({
                 <Button event="Page Receive Step 3 re-verify" outlineGrey onClick={onVerify}>
                   <Trans i18nKey="common.reverify" />
                 </Button>
-                <Button id={"receive-receive-continue-button"} primary onClick={onClose}>
+                <Button data-test-id={"receive-receive-continue-button"} primary onClick={onClose}>
```
[the remark this answers is not in the document]
Comment (nabil-brn) [reply src/renderer/modals/Receive/steps/StepReceiveFunds.js:217]: Is there a reason why the id starts with "receive-receive-..." ?

**Review by nabil-brn — DISMISSED** [review]

**Review by valpinkman — APPROVED** [review]

**Pull request merged**